### ISSUE TITLE
[BUG] Gallery page

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/gbowne1site_react#37]
The gallery css needs some work.

**Comment by web-dev-kushal** [comment]
Hey, I would likre to work on this issue

**Comment by gbowne1** [comment]
@web-dev-kushal let me know what you need help with in this issue.

**Comment by web-dev-kushal** [comment]
Yes I will start working...Please do assign me

**Comment by gbowne1** [comment]
Ok, @web-dev-kushal

**Comment by web-dev-kushal** [comment]
how do you want the gallery page ...can you show me an example template

**Comment by gbowne1** [comment]
@web-dev-kushal It needs a lot of work.. this is how it was before something got messed up.. 

It's mentioned here: https://github.com/gbowne1/gbowne1site_react/pull/27

**Comment by web-dev-kushal** [comment]
the navbar and all are messed up...I think with the styling and all

**Comment by gbowne1** [comment]
yeah. I made a separate issue for that.  not been able to track down what did that.

**Comment by gbowne1** [comment]
It's really wierd because, not much really changed that much since it was working properly.

**Comment by gbowne1** [comment]
I never really settled on a design either.  My idea was being able to post pics of my numerous projects and post a little bit about each one.
